keras_nlp/models/bloom/bloom_backbone.py

```diff
@@ -105,7 +105,6 @@ def __init__(
             input_dim=vocabulary_size,
             output_dim=hidden_dim,
             embeddings_initializer=_bloom_kernel_initializer(stddev=0.02),
-            tie_weights=False,
```

Comment:
does this mean we should update our test preset here? https://www.kaggle.com/models/keras/bloom

Reply:
Yes That's right, because the `reverse_embeddings` weights will be there for no reason. I didn't notice it. Thanks!

Reply:
The Model parameters dropped from 816m to 560m which is the model size, I should have noticed that from the beginning. :smiley:

Reply:
checkout here :
https://www.kaggle.com/models/mohamedabuelnasr/bloom

I have uploaded bloom & bloomz 560m, 1b1, 1b7, 3b

Reply:
Thanks! Pulling `bloom_560m_multi` into keras right now so we have something to test.

Is float16 the dtype used for the source weights? (if so sounds good to me!)

Re naming, can we do `1.1b` intead of `1b1`? We just already have that conversion, e.g. `opt_1.3b_en`. Thanks!

Reply:
Yes I think that the model is stored in float16. I didn't find it mentioned in the model card, but the weights file size is the same as the hf weights file size. 
I also checked it numerically by multiplying the number of weights by 2 (2 bytes for float16) and dividing by a gigabyte and this should produce the file size for hf and keras weights.


Re naming, The reason I choosed that naming at first is because of the opt preset naming actually. But I don't think it's good to have a (dot) inside a name, the name will never be copied using a double click. That might be silly, but that is the reason I changed the name. Only to make it copiable using a double click. 
Anyways, I will change the preset name to the old name, unless you change your mind.

Reply:
Yeah, makes sense, but given that OPT is already released, let's stick with what we got. We could consider renaming all our presets names to avoid dot, but let's not fragment. Basically, better to be consistent with imperfect naming, than inconsistent.

And just perusing https://huggingface.co/models, seems like a dot in a name like this won't be too shocking to general user expectations. I count five of 30 with a dot somewhere while writing this.